sunpy/map/sources/soho.py

```diff
@@ -105,13 +121,25 @@ class LASCOMap(GenericMap):
     ----------
     * `SOHO Mission Page <https://sohowww.nascom.nasa.gov/>`_
     """
-
     def __init__(self, data, header, **kwargs):
         super().__init__(data, header, **kwargs)
 
         self.plot_settings['cmap'] = f'soholasco{self.detector[1]!s}'
         self.plot_settings['norm'] = ImageNormalize(
             stretch=source_stretch(self.meta, PowerStretch(0.5)), clip=False)
+        
+    @property
+    def coordinate_system(self):
+        """
+        Override the default implementation to handle LASCOMAP-specific logic for CTYPE values.
+        """
+        ctype1 = self.meta.get('ctype1', None)
```

Comment:
What happens if meta is none? Would the if line not error?

Reply:
yea it would. WIll add a if self.meta condition for the same